docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -131,6 +131,9 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -Xmx16G
     -XX:InitialRAMPercentage=80
     -XX:MaxRAMPercentage=80
+    -XX:+UnlockDiagnosticVMOptions
+    -XX:+UseAESCTRIntrinsics
```

Comment:
I would prefer to keep GC options together, and AES intrinsics isn't one of them.
I wasn't aware `-XX:+UnlockDiagnosticVMOptions` needs to come first.
Let's revert to your original order, where you simply added the new option as the last.